react/features/polls/components/web/PollCreate.tsx

```diff
@@ -206,11 +206,11 @@ const PollCreate = ({
                     value = { question } />
             </div>
             <ol className = { classes.answerList }>
-                {answers.map((answer: string, i: number) => {
+                {answers.map((answer, i: number) => {
 
                     const isIdenticalAnswer = answers.slice(0, i).length === 0 ? false
-                        : answers.slice(0, i).some((prevAnswer: string) =>
-                            prevAnswer === answer && prevAnswer !== '' && answer !== '');
+                        : answers.slice(0, i).some(prevAnswer =>
+                            prevAnswer === answer && prevAnswer.name !== '' && answer.name !== '');
```

Comment:
you should prolly check prevAnswer.name === answer.name instead of object comparison which is always false